wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceService.java

```diff
@@ -656,6 +667,41 @@ public void deleteProject(@ApiParam("The workspace id")
         }
     }
 
+    @GET
+    @Path("/{id}/check")
+    @Produces(APPLICATION_JSON)
+    @ApiOperation(value = "Get state of the workspace agent by the workspace id and agent id")
+    @ApiResponses({@ApiResponse(code = 200, message = "The response contains requested workspace entity"),
+                   @ApiResponse(code = 400, message = "Missed required parameters, parameters are not valid"),
+                   @ApiResponse(code = 403, message = "The user is not workspace owner"),
```

Comment:
They are users who also can use IDE but they are not owners of WS, so description looks not complete